rpcs3/Emu/Cell/Modules/cellAudio.cpp

```diff
@@ -790,15 +791,6 @@ void cell_audio_thread::mix(float *out_buffer, s32 offset)
 					out_buffer[out + 0] = left;
 					out_buffer[out + 1] = right;
 
-					if constexpr (!DownmixToStereo)
```

Comment:
Isn't this there for when you have surround but the game doesn't support it? Not sure if the buffer is zero-initialized or if the backend is enforced to match the number of channels selected by the game so would be nice to clarify.

Reply:
port.num_channels == 2 and constexpr u32 channels = DownmixToStereo ? 2 : 8 in cell_audio_thread::mix won't ever allocate enough buffers for this to make sense since this: for (u32 out = 0, in = 0; out < out_buffer_sz; out += channels, in += 2) will never be more than two so i think it is safe to remove that unnecessary bit here